types/node/buffer.buffer.d.ts

```diff
@@ -160,6 +160,26 @@ declare module "buffer" {
              * @param encoding The encoding of `string`. **Default:** `'utf8'`.
              */
             from(string: WithImplicitCoercion<string>, encoding?: BufferEncoding): Buffer<ArrayBuffer>;
+            /**
+             * Creates a view of the `ArrayBuffer` or a new `Buffer` containing `string`.
+             * The `encoding` parameter identifies the character encoding to be used when
+             * converting `string` into bytes.
+             *
+             * A `TypeError` will be thrown if `string` is not a string or another type
+             * appropriate for `Buffer.from()` variants.
+             *
+             * `Buffer.from(string)` may also use the internal `Buffer` pool like
+             * `Buffer.allocUnsafe()` does.
+             * @since v5.10.0
+             * @param arrayBufferOrString An array buffer or a string to encode.
+             * @param offsetOrEncoding A byte-offset or encoding.
+             * @param length A length.
+             */
+            from(
+                arrayBufferOrString: WithImplicitCoercion<ArrayLike<number> | string>,
+                offsetOrEncoding?: number | BufferEncoding,
+                length?: number,
```

Comment:
```suggestion
                encoding?: BufferEncoding,
```
Should be fine to have this as an optional parameter. It'll just be ignored if an array-like is passed.

The third parameter only applies when passing array buffers, so isn't relevant.